src/cmdlib.sh

```diff
@@ -288,6 +287,45 @@ commit_overlay() {
         --skip-list <(echo /statoverride)
 }
 
+create_content_manifest(){
+    mkdir -p "${workdir}"/tmp/buildinfo
+    base_repos=$(jq .repos "${manifest_tmp_json}")
+    # Make an empty json file in tmp/buildinfo/
+    echo {} > "$workdir"/tmp/buildinfo/content_manifest.json
+
+    # Get the data form content_sets.yaml and map the repo names given in '$base_repos' to their corresponding
+    # pulp repository IDs provided in https://www.redhat.com/security/data/metrics/repository-to-cpe.json
+    python3 -c "
+import json, yaml;
+# Open the yaml and load the data
+f = open('$1')
+data = yaml.safe_load(f);
+f.close();
+repos=[];
+
+for baseRepo in $base_repos:
+    if (baseRepo in data['repo_mapping'].keys()):
+        for i in data['repo_mapping'][baseRepo]:
+            if(i=='name'):
+                data['repo_mapping'][baseRepo][i]=data['repo_mapping'][baseRepo][i].replace('\$ARCH', '$(arch)');
+                repos.append(data['repo_mapping'][baseRepo][i])
```

Comment:
Ideally here we'd be checking that the baseurls match. This would require parsing the repo files though. We can skip this for now.